.env.development

```diff
@@ -1,5 +1,5 @@
 ENV=dev
-
+PROBATE_FEES_RELEASE_DATE=2017-05-01
```

Comment:
I think that you should set this to different data so we can test it :) after it's deployed, so let say Monday next week?